password-generator-app/password.css

```diff
@@ -0,0 +1,254 @@
+*{
+ box-sizing: border-box;
+}
+
+body {
+  background: #18171F;
+  font-family: 'JetBrainsMono-Bold';
+}
+
+@font-face {
+  font-family: 'JetBrainsMono-Bold';
+  src: url("./assets/fonts/static/JetBrainsMono-Bold.ttf");
+}
+
+main {
+ display: flex;
+ flex-direction: column;
+ justify-content: center;
+ align-items: center;
+flex-wrap: wrap;
+height: 90vh;
+width:100% ;
+text-align: center;
+color: white;
+  
+}
+
+.title {
+  font-size: 24px;
+  font-weight: bold;
+  margin-bottom: 20px;
+  color: #817D92;
+}
+
+.top {
+  width: 100%;
+ 
+  margin-bottom: 20px;
+  background: #24232C;
+  box-sizing: border-box;
```

Comment:
box sizing usually applies to all elements

Reply:
Well noted.

Reply:
> box sizing usually applies to all elements

Well noted